ProcessHacker/hndlprp.c

```diff
@@ -1111,10 +1111,34 @@ VOID PhpUpdateHandleGeneral(
                 PhDereferenceObject(name);
             }
 
+            NtQueryInformationThread(dupHandle,
+                ThreadIsTerminated,
+                &isTerminated,
+                sizeof(BOOL),
```

Comment:
sizeof(ULONG)

Reply:
Wouldn't it be easier to use `sizeof isTerminated`? One has to just change the variable declaration.

Reply:
@Biswa96, For some reason, this approach is hardly ever used throughout the codebase. Don't know why, though.

Reply:
@Biswa96 @diversenok It makes it easier for anyone (even without experience) to review the calls 
 and for consistency.